pycfmodel/model/resources/iam_role.py

```diff
@@ -51,3 +52,12 @@ def policy_documents(self) -> List[OptionallyNamedPolicyDocument]:
         for policy in policies:
             result.append(OptionallyNamedPolicyDocument(name=policy.PolicyName, policy_document=policy.PolicyDocument))
         return result
+
+    @property
+    def all_statement_conditions(self) -> List[StatementCondition]:
+        assume_role_policy_document_conditions = [
+            statement.Condition
+            for statement in self.Properties.AssumeRolePolicyDocument.statement_as_list()
+            if statement.Condition
+        ]
```

Comment:
just thinking that this should be a property/method itself
while this property (`all_statement_conditions`) keep the return as it is, but returning both properties as one
what do you think?

Reply:
See 61b14e8